### ISSUE TITLE
rgba not working 

### ISSUE THREAD

**Issue by jariwalabhavesh** [Bhoos/react-native-theming#3]
Great Library and pretty use full as well,

I have one issue which i am not able to figuring out how to do

Theme File
```
const AppThemes = [
  createTheme({
    backgroundColor: '#FFFFFF',
   rgbaBackground: '255,255,255'
  }, 'Theme1'),
  createTheme({
    backgroundColor: '#222222',
   rgbaBackground: '0,0,0'
  }, 'Theme1'),
];
```

For example: I have backgroundColor in css file.

```
 button: {
    backgroundColor: '@backgroundColor',
  }
```
It is working fine with above code.

But if i try to use rgba Value like:

```
button:{
    backgroundColor: 'rgba(@rgbaBackground,0.2)',
}

```
which is giving error. I want to keep opacity dynamic only need rgb value of base color from theme.

Any idea how can i do that????

**Comment by syaau** [comment]
@jariwalabhavesh I have fixed this issue in version 1.0.10.

**Comment by jariwalabhavesh** [comment]
@syaau 
Thanks for quick fix.
I had tested it and it is working like charm.
